include/roboteam_ai/world_new/World.hpp

```diff
@@ -37,10 +38,19 @@ class World {
    public:
     /**
      * Global singleton for World, scott-meyers style
+     * @param[in] resetWorld Boolean that marks whether to reset the world before returning
+     * if set to true, world.clear() is called
+     * if set to false, worldInstance is simply reset
      * @return A pointer to a static World
      */
-    inline static World *instance() {
+    inline static World *instance(bool resetWorld = false) {
         static World worldInstance{&rtt::SETTINGS};
+        if (resetWorld) {
+#ifndef RUNNING_TEST
```

Comment:
if we are using macro's anyways, why not check if there is a macro that indicates a google test environment and completely enable/disable the function with this macro? :-)

Reply:
true, will add